src/v/rpc/types.h

```diff
@@ -192,6 +192,7 @@ class netbuf {
     void set_compression(rpc::compression_type c);
     void set_service_method_id(uint32_t);
     void set_min_compression_bytes(size_t);
+    void set_version(transport_version v) { _hdr.version = v; }
```

Comment:
Needs to be in the previous commit?

Reply:
wow nice catch. not sure how I missed that one